src/styles/main/_index.scss

```diff
@@ -5,6 +5,7 @@
 @import "filter";
 @import "building-report";
 @import "year-control";
+@import "toolbar_dropdown";
```

Comment:
sigh. Why does the codebase have dashes someplaces and underscores others. Anyway, just use the same as what's around what you're introducing.